securedrop_client/locale/bn/LC_MESSAGES/messages.po

```diff
@@ -0,0 +1,319 @@
+# Translations template for SecureDrop Client.
+# Copyright (C) 2022 Freedom of the Press Foundation
+# This file is distributed under the same license as the SecureDrop Client project.
+# FIRST AUTHOR <EMAIL@ADDRESS>, 2022.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: SecureDrop Client 0.7.0\n"
+"Report-Msgid-Bugs-To: securedrop@freedom.press\n"
+"PO-Revision-Date: 2022-08-19 14:39+0000\n"
+"Last-Translator: diyaf <diyaf70383@yubua.com>\n"
+"Language-Team: Bengali <https://weblate.securedrop.org/projects/securedrop/securedrop-client/bn/>\n"
+"Language: bn\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=n > 1;\n"
+"X-Generator: Weblate 4.13\n"
+"Generated-By: Babel 2.10.1\n"
+
+msgid "{application_name} is already running"
+msgstr "{application_name} চলমান"
+
+msgid "The SecureDrop server cannot be reached. Trying to reconnect..."
+msgstr "SecureDrop সার্ভারে পৌছানো যাচ্ছে না। পুনরায় চেষ্টা করা হচ্ছে..."
```

Comment:
Oh, we're gotta have to make sure we've got font support.

![Incorrect rendering of Bengali text.](https://user-images.githubusercontent.com/1619067/186529563-46f04dcb-9703-4ab2-85c0-701406a050fa.png)

Reply:
Thank you Gonzalo, you're right we should make sure we have font support! Adding this + localization guidelines in the UX wishlist

Reply:
Note for future reference: the [Noto](https://en.wikipedia.org/wiki/Noto_fonts) family looks very promising to me.